composer.json

```diff
@@ -51,6 +52,12 @@
             "Acquia\\Drupal\\RecommendedSettings\\": "src/"
         }
     },
+    "repositories": {
```

Comment:
@rajeshreeputra Looks like this is also not required.

Reply:
sure, as discussed, removed.